app/src/main/java/io/github/wulkanowy/ui/modules/timetablewidget/TimetableWidgetConfigureActivity.kt

```diff
@@ -38,11 +39,26 @@ class TimetableWidgetConfigureActivity : BaseActivity<TimetableWidgetConfigurePr
     }
 
     override fun initView() {
-        widgetConfigureRecycler.apply {
+        with(widgetConfigureRecycler) {
             adapter = configureAdapter
             layoutManager = SmoothScrollLinearLayoutManager(context)
         }
-        configureAdapter.setOnItemClickListener { presenter.onItemSelect(it) }
+
+        configureAdapter.setOnItemClickListener(presenter::onItemSelect)
+    }
+
+    override fun showThemeDialog() {
+        val items = arrayOf(
+            getString(R.string.widget_timetable_theme_light),
+            getString(R.string.widget_timetable_theme_dark)
+        )
+
+        AlertDialog.Builder(this, R.style.WulkanowyTheme_WidgetAccountSwitcher)
+            .setTitle(R.string.widget_timetable_theme_title)
+            .setSingleChoiceItems(items, 0) { _, which ->
```

Comment:
Don't set default item

```suggestion
            .setSingleChoiceItems(items, -1) { _, which ->
```

> * @param checkedItem specifies which item is checked. If -1 no items are checked.